students/MicahBraun/Lesson 03/mailroom.py

```diff
@@ -0,0 +1,227 @@
+# ------------------------------------------------------------------------
+# NAME: MICAH BRAUN
+# PROJECT: mailroom.py
+# PURPOSE: Working with arraysv(nested lists), manipulating data
+# DATE: 05/24/2018
+#
+# DESCRIPTION: Program is menu-driven user interface for interacting
+#              with a nested list of stored information. User is able
+#              to add names to a list (not currently saved upon program
+#              exit), append values to fields, check against current
+#              fields, create mock "Thank You" e-mails based on user-
+#              -input and associated fields, and generate tabular display
+#              of data for viewing.
+# ------------------------------------------------------------------------
+
+# Data -------------------------------------------------------------------
+spacing = '- ' * 55  # formatting for DONOR header
+data_list = [['NAMES', 'DONATION AMOUNT', 'NUMBER OF GIFTS', 'AVG. GIFTS'],
```

Comment:
don't store "header" as part of data structure, it will only complicate your code